src/components/chat/TopBar.tsx

```diff
@@ -15,26 +15,43 @@ function TopBar({ teamName }: props) {
   const weekList = ['일', '월', '화', '수', '목', '금', '토'];
   const day = weekList[dayIndex];
 
+  const range =
+    index == 0 || index == 1
+      ? 0
+      : index % 2 == 0
+      ? Math.floor(((Math.floor(index / 2) - 1) / 10) * 100)
+      : Math.floor((Math.floor(index / 2) / 10) * 100);
```

Comment:
index == 1 은 조건에 없어도 결과값이 제대로 나오는 것 같아요!

가독성을 위해 다중 삼항 연산자를 지양하는 편이라서 이렇게 작성해봐도 좋을 것 같은데,
혹시 Math.floor 연산자를 따로 사용한 이유가 있을까요??
```suggestion
const range = (index) => {
    if (index === 0) {
        return 0;
    }
    return (index % 2 === 0 ? (index - 2) : (index - 1)) / 2 * 10 ;
}
```

Reply:
악 저 index==1  지운다는걸 까먹구 그대로 넣어부렸네... (머쓱) 나는 계산을 소수점을 버리고(여기서 math.floor를 쓰고) 그거를 가지고 /2 하는 방식으로 해줬는데 유진언니가 개선해준 방법이 훨 좋은거 같아!!